src/android/SplashScreen.java

```diff
@@ -194,9 +263,25 @@ public void run() {
                 // TODO: Use the background color of the webView's parent instead of using the
                 // preference.
                 root.setBackgroundColor(preferences.getInteger("backgroundColor", Color.BLACK));
+                
                 root.setLayoutParams(new LinearLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT,
                         ViewGroup.LayoutParams.MATCH_PARENT, 0.0F));
-                root.setBackgroundResource(drawableId);
+                
+                // Use an ImageView to render the image because of its flexible scaling options.
+                splashImageView = new ImageView(context);
+                splashImageView.setImageResource(drawableId);
```

Comment:
I am trying to test this, but I don't ever see the background color. On both my Samsung Galaxy S3 and on a Genymotion 4.1.1 emulator the image resizes too quickly to see BG. Do you have a way of testing background color?